spec/ruby_llm/providers/anthropic/tools_spec.rb

```diff
@@ -68,7 +68,7 @@
         instance_double(Message,
                         content: '',
                         tool_calls: {
-                          'tool_123' => instance_double(ToolCall,
+                          'tool_123' => instance_double(RubyLLM::ToolCall,
                                                         id: 'tool_123',
                                                         name: 'test_tool',
                                                         arguments: { 'arg1' => 'value1' })
```

Comment:
I think those tests were wrongly relying on the Dummy app `ToolCall` class VS the `RubyLLM` one